BackEnd/src/main/kotlin/quest/marketstack/tradingapp/TradingAppApplication.kt

```diff
@@ -1,4 +1,4 @@
-package quest.marketstack.TradingApp
+package quest.marketstack.tradingapp
```

Comment:
Package/directory names (with the exception of the highest level folder, i.e: BackEnd) should always be fully lowercase. So whenever you're making a new folder, make sure everything is lowercase.